### PR TITLE
Allow checking for release when project is created/edited

### LINKED ISSUE #333: RFE: Check versions immediately after new project is added
Hi, I'd like to be able to add projects programatically and then immediately query them for latest versions by the `by_ecosystem` endpoint. IIUC, right now I'd need to:
- create the project
- query it by the `by_ecosystem` endpoint to get its ID
- use the `/api/version/get` to trigger getting newest versions and get it

I'd like to be able to get version info from the `by_ecosystem` endpoint right after I add the project, so this would mean adding `check_release` to the UI endpoint that adds the new project. Does this make sense? I can send a PR if you think it's worth implementing.

### PR DESCRIPTION
**Pull request by bkabrda**
- update project form, implement functionality in ui and template
- add ui tests
- add rpm-python to Dockerfile, since it's necessary for check_release function

Fixes #333.

### CODE REVIEW

File: tests/test_flask.py [hunk 1]
```diff
@@ -330,7 +331,46 @@ def test_new_project(self):
...
-    def test_edit_project(self):
+    @mock.patch('anitya.check_release')
+    def test_new_project_with_check_release(self, patched):
+        output = self.app.get('/project/new', follow_redirects=True)
+        with anitya.app.APP.test_client() as c:
+            with c.session_transaction() as sess:
+                sess['openid'] = 'openid_url'
+                sess['fullname'] = 'Pierre-Yves C.'
+                sess['nickname'] = 'pingou'
+                sess['email'] = 'pingou@pingoured.fr'
```
Comment (pypingou) [inline tests/test_flask.py]: note for later, make it a little harder for spammer to find this one ^^

File: tests/test_flask.py [hunk 2]
```diff
@@ -330,7 +331,46 @@ def test_new_project(self):
...
+                'csrf_token': output.data.split(
+                    b'name="csrf_token" type="hidden" value="')[1].split(b'">')[0],
+            }
+            output = c.post(
+                '/project/new', data=data, follow_redirects=True)
+            self.assertEqual(output.status_code, 200)
+            self.assertTrue(
+                b'<li class="list-group-item list-group-item-default">'
+                b'Project created</li>' in output.data)
+            patched.assert_not_called()
```
Comment (pypingou) [inline tests/test_flask.py]: oh, cool one :)

File: tests/test_flask.py [hunk 3]
```diff
@@ -415,6 +455,26 @@ def test_edit_project(self):
...
+            data['check_release'] = 'on'
+            output = c.post(
+                '/project/3/edit', data=data, follow_redirects=True)
+            self.assertEqual(output.status_code, 200)
+            self.assertTrue(
+                b'<li class="list-group-item list-group-item-default">'
+                b'Project edited</li>' in output.data)
+            patched.assert_called_once_with(mock.ANY, mock.ANY)
+
+
```
Comment (pypingou) [inline tests/test_flask.py]: pep8 will likely ask for one 1 line here

**Comment by pypingou** [comment]
Looking good, do you want to address the styling or shall I merge?

**Comment by bkabrda** [comment]
I removed the extra empty line in one of the tests and force-pushed the change (not really worth making a commit IMO), so this is ready to merge I think. Thanks for reviewing!

**Comment by pypingou** [comment]
Yes, it is, thanks! :)

**Pull request merged**